### DISCUSSION TITLE
TrailBlazor Conference

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4506] [General]
I mentioned this in the Oqtane Developer Meeting this week.... we will be partnering with C# Corner to host a Blazor/Oqtane virtual conference on Thu, Oct 3. The conference will be branded "TrailBlazor" and Daniel Roth has agreed to be a keynote speaker. We will be conducting a call for speakers in the coming week.

**Comment by leigh-pointer** [comment]
@sbwalker You might want to update the Topic Title ;)

**Comment by markdav-is** [comment]
looking forward to putting together a talk proposal.  I'd like to do one on cross-module coms and borrowing concepts form the game industry to build engaging experiences.

**Reply by sbwalker** [reply]
@markdav-is I haved added you as a speaker. If you want a different bio/headshot please share in this thread.

**Reply by markdav-is** [reply]
thanks.  I'll check it out!

**Reply by markdav-is** [reply]
here's some headshots that don't have the AI goofiness.
![BD5-11-15-headshot](https://github.com/user-attachments/assets/b8b7f95c-b958-4f8c-a1ff-3f17813f3234)
![mark-work-bg](https://github.com/user-attachments/assets/9fa61562-0a0c-48c1-92cc-1678457d8dc7)

**Reply by sbwalker** [reply]
@markdav-is I updated your photo

**Reply by markdav-is** [reply]
thanks so much!  You can remove the "AI strategy" bits from my profile.  That's an inside joke with my linked-in friends.

**Comment by ADefWebserver** [comment]
I plan to submit a proposal on integrating components like PowerBI and Syncfusion

**Reply by sbwalker** [reply]
@ADefWebserver this would be perfect - especially the Syncfusion integration - I have added you as a speaker.

**Comment by markdav-is** [comment]
@sbwalker any new updates on TrailBlazor conf?  Is the URL we should be checking https://[trailblazor.net](https://trailblazor.net/)/ ?

**Comment by sbwalker** [comment]
@markdav-is  yes the conference site will be at https://trailblazor.net however there has not been a public announcement yet as I just finished getting the registration form and content published today.

**Reply by nicpitsch** [reply]
@sbwalker I have registered using the registration form at https://trailblazor.net. It's days since then but I never got an e-mail confirmation as the form response says. My e-mail address has a hyphen within, so I tried with another one, again without success. I have spam checked about trailblazor.net and devessence.com e-mail's. Could the ch-domain be a cause?

**Reply by sbwalker** [reply]
@nicpitsch thank you for letting me know. I investigated the problem and it appears that there was a record with a corrupted email address causing an issue with email delivery. This should be resolved now.

**Reply by nicpitsch** [reply]
Thanks, I can confirm the e-mail confirmation sended by sendgrid.net.

**Comment by sbwalker** [comment]
The TrailBlazor Conference (https://trailblazor.net) has quite a few speakers confirmed now - I am hoping for 1 more from Microsoft to talk about Blazor Hybrid.

**Comment by hishamco** [comment]
Can we talk about Localization in Oqtane Framework what we have, and the future of localization in Oqtane?

**Reply by hishamco** [reply]
@sbwalker seems no interested in localization using Blazor ;)

**Reply by sbwalker** [reply]
@hishamco my apologies for not answering sooner. Since this is a one day event there are limited speaking slots. I already extended invitations to a number of speakers and I am still waiting for confirmations. If there are any slots remaining by next week I will let you know.

**Reply by hishamco** [reply]
No problem Shaun, could you please let me know the sessions titles, hopefully I get time to watch the event

**Comment by sbwalker** [comment]
I just got confirmation from Beth Massi that she will do a session on Blazor Hybrid :)

**Comment by sbwalker** [comment]
The conference schedule as well as all session titles and abstracts are now published on https://trailblazor.net (except for @iJungleboy )

**Reply by iJungleboy** [reply]
Just sent you my abstract, sorry for the delay... ;)

**Comment by thabaum** [comment]
@sbwalker can you edit the original post with information on exactly where and when we will be able to stream this live?  I would love to set this in my calendar without having to look it all up again and the top post I was hoping to see these details.  Thanks!

**Reply by thabaum** [reply]
Also I forgot my manors with a please and thank you for setting this up for us all!

**Reply by thabaum** [reply]
An idea if not already posted would be a link to a blog post on the oqtane.org website with details would be pretty nice as it is very search engine optimized and would help drive even more traffic.

**Comment by sbwalker** [comment]
All of the recordings from the conference are now posted:

https://trailblazor.net/

**Reply by thabaum** [reply]
@sbwalker this is great, thanks for adding these videos.  

Question:  Is there a way for you to limit how many commercials we get in the video?  Every 5 minutes is a commercial or two, it is so annoying I don't want to watch these videos. Not sure if it is a channel preference or YouTube being greedy.  I miss the old YouTube before Google bought it out... it was so much nicer to use and watch videos on with less commercials.  It is currently a horribly bad experience.  Every 15 minutes maybe... but every 5 is just killing my train of thought with each discussion.

**Reply by thabaum** [reply]
https://www.c-sharpcorner.com/videos/trail+blazor

it would be nice to find it here since they are a sponsor of the event.  is there a way to get a channel going here?

**Reply by sbwalker** [reply]
@thabaum there is no way to prevent advertising on YouTube. Google bought YouTube in 2006... so I am not sure I even remember what it was like 18 years ago - but I agree that the advertising has become more disruptive to the experience over time. I believe that the recording will also be published by CSharp Corner... I am guessing they just have not had time yet.

**Comment by ADefWebserver** [comment]
I watch so much YouTube that [YouTube Premium ](https://www.youtube.com/premium) is worth it (no commercials)

**Reply by thabaum** [reply]
It is annoying, only way to enjoy it is to pay I get it, which I don't watch much and dont want to support that business as it has scroogled mine with search results in the past massively.  Google can go fly a kite!

**Reply by thabaum** [reply]
In fact Google has sent my state checks to pay my business, instead of directly to me.  I wont even collect them... again I really can't stand bullies and monopolies.  It really makes all this development time worthless if they control winners/losers, what is the point of being creative.  I just really work with Oqtane anymore and give my work away for you all to enjoy for the fun of it really.

**Comment by thabaum** [comment]
.NET 9 looks pretty cool with new scaffolding.  I did manage to fight through watching half of Daniel Roth's presentation.  I will try to finish it today.  I want to watch the following as well as those provided by the Oqtane community.  Thank you all for your generous contributions sharing.  Sounds like a few issues discussed prior to his presentation helped him out with some of the community concerns.  I will have to make time to watch that as well.
